examples/01_cardio_tiny_dataset/run_example.sh

```diff
@@ -62,11 +62,11 @@ WF_ID=`fractal --batch workflow new "$WF_NAME" $PRJ_ID`
 echo "WF_ID: $WF_ID"
 
 # Add tasks to workflow
-fractal workflow add-task $WF_ID "Create OME-Zarr structure" --args-file Parameters/args_create_ome_zarr.json --meta-file Parameters/example_meta.json
+fractal workflow add-task $WF_ID "Create OME-Zarr structure" --args-file Parameters/args_create_ome_zarr.json --meta-file Parameters/example_meta_gpu.json
```

Comment:
Let's not ask for a GPU node for the create ome zarr task. Doesn't make much sense content-wise. => let's just drop the meta parameters when adding this task

Reply:
I am not sure of why this ended up here, it was for sure a spurious commit. I will remove it.

Reply:
(in any case, this use of meta JSON files should be made obsolete by the appropriate defaults coming from fractal-tasks-core 0.9.1 or higher)

Reply:
> (in any case, this use of meta JSON files should be made obsolete by the appropriate defaults coming from fractal-tasks-core 0.9.1 or higher)

Yes, they wouldn't be needed anymore for the example to run. Would be good if one example still contained the usage of the meta-parameters in their new way, e.g. changing cpu & mem requirements for a task. Not because it would be needed for that task to run, but to provide an example of how it is done to users :)